js/components/close-caption-multi-audio-menu/closeCaptionMultiAudioMenu.js

```diff
@@ -0,0 +1,145 @@
+var React = require('react');
+var CloseCaptionTab = require('./closeCaptionTab');
+var MultiAudioTab = require('./multiAudioTab');
+
+var CloseCaptionMultiAudioMenu = React.createClass({
+
+  getInitialState: function () {
+    this.multiAudio = {};
+    this.closeCaptions = {};
+
+    if(this.props.controller && this.props.controller.state) {
+      this.multiAudio = this._getMultiAudio();
+      this.closeCaptions = this._getClosedCaptions();
+    }
+
+    return {
+      multiAudio: this.multiAudio,
+      closeCaptions: this.closeCaptions,
+      skinConfig: this.props.skinConfig
+    }
+  },
+
+  /**
+   * Getting a list of available audio tracks
+   * @return {object} - list of available audio tracks and selected track
+   * @private
+   */
+  _getMultiAudio: function () {
+    var multiAudio = {
+      list: [],
+      selected: null
+    };
+
+    if(!!this.props.controller.state.multiAudio) {
+      if(Array.isArray(this.props.controller.state.multiAudio.tracks)) {
+        multiAudio.list = this.props.controller.state.multiAudio.tracks;
+        multiAudio.selected = _.find(this.props.controller.state.multiAudio.tracks, function (track) {
+          return track.enabled === true;
+        });
+      }
+    }
+
+    return multiAudio;
```

Comment:
What is the reason behind copying the state from the props to the local state? If the props are modified outside of this component then it won't react to these changes. Unless there's a reason to duplicate this state, we should consider using the props directly.